superset/config.py

```diff
@@ -366,6 +366,8 @@ def _try_json_readsha(  # pylint: disable=unused-argument
 # always be the table layout
 if DEFAULT_FEATURE_FLAGS["THUMBNAILS"]:
     DEFAULT_FEATURE_FLAGS["LISTVIEWS_DEFAULT_CARD_VIEW"] = True
+else:
+   DEFAULT_FEATURE_FLAGS["LISTVIEWS_DEFAULT_CARD_VIEW"] = False 
```

Comment:
nit: this is equal to
```
DEFAULT_FEATURE_FLAGS["LISTVIEWS_DEFAULT_CARD_VIEW"] = DEFAULT_FEATURE_FLAGS["THUMBNAILS"]
```
Does it still make sense to have this feature flag?

Reply:
I think we actually don't need it. As we can do the conditional check using the thumbnails feature flag

Reply:
This feature flag was added as per @bkyryliuk so that it can default to a certain view, regardless if thumbnails are enabled or not

Reply:
yeah - this is quite needed. Setting up thumbnails is quite complex and has some security implications. Having this feature flag helps to default to the list view for the superset users when  thumbnails are not enabled